Assets/Plugins/iOS/Shopify/MessageCenter.swift

```diff
@@ -32,14 +32,14 @@ import PassKit
     private static var messages = [String: UnityMessage]()
     
     static func send(_ message: UnityMessage) {
-        UnityInterfaceWrapper.sendMessage(try! message.serializedString(), toObject: message.recipientObjectName, havingMethodName: message.recipientMethodName)
+        MessageSender.sendMessage(try! message.serializedString(), object: message.object, method: message.method)
     }
     
     static func send(_ message: UnityMessage, completionHandler: UnityMessage.MessageCompletion?) {
         
         messages[message.identifier] = message
         
-        UnityInterfaceWrapper.sendMessage(try! message.serializedString(), toObject: message.recipientObjectName, havingMethodName: message.recipientMethodName)
+        MessageSender.sendMessage(try! message.serializedString(), object: message.object, method: message.method)
```

Comment:
I believe that since `MessageSender` is included in the `Unity-iPhone-Bridging-Header.h` we would be unable to include the Swift bridging header here. And `UnityMessage` is defined in Swift. `MessageSender` is really just an Objective-C wrapper, but named as `MessageSender` to make it more clear of what it is doing